pkg/tsdb/influxdb/flux/builder.go

```diff
@@ -207,7 +209,11 @@ func (fb *frameBuilder) Init(metadata *query.FluxTableMetadata) error {
 	if hasSimpleTimeCol && (len(nonTimestampCols) == 1) && (nonTimestampCols[0].name == "_value") {
 		// there is a simple timestamp column, and there is a single non-timestamp value column
 		// named "_value". we decide that this is "the" value-column.
-		nonTimestampCols[0].isTheSingleValue = true
+		nonTimestampCols[0].isTheSimpleValue = true
+
+		// now that we know that there is both a single correctly named timestamp column
```

Comment:
I wonder if it wouldn't make more sense to have these comments where the field is defined in the struct to explain what the field and true/false values mean in a single place.

Reply:
thanks, i added a comment to that place for now, later maybe some comment-refactoring will be needed :)